.github/workflows/test.yml

```diff
@@ -84,6 +84,9 @@ jobs:
           sudo apt-get install -y --no-install-recommends clang-16 lld-16 libunwind-16 libc++abi1-16 libc++1-16 libc++-16-dev libclang-rt-16-dev
           echo "build:linux --action_env=CC=/usr/lib/llvm-16/bin/clang --action_env=CXX=/usr/lib/llvm-16/bin/clang++" >> .bazelrc
           echo "build:linux --host_action_env=CC=/usr/lib/llvm-16/bin/clang --host_action_env=CXX=/usr/lib/llvm-16/bin/clang++" >> .bazelrc
+          echo "build:linux --copt='-Werror'" >> .bazelrc
+          echo "build:linux --copt='-Wno-error=#warnings'" >> .bazelrc
+          echo "build:linux --copt='-Wno-error=deprecated-declarations'" >> .bazelrc
```

Comment:
Do we want need this if we have .bazelrc:47 `build --per_file_copt='external/v8@-Wno-deprecated-declarations'`
It's possible both are required, just not sure.

Reply:
This would handle a project other than v8 having deprecated declarations.

Reply:
Yeah that's why I asked if we want this in any project other than v8.
I guess it's a more fitting question for @jasnell  maybe?

Reply:
I think we regularly have deprecation warnings for JSG calling deprecated V8 functions (including right now?), it will be difficult to always have those under control.

Reply:
But these would fall under .bazelrc:47 no?

Reply:
No, that directive only adds the `-Wno-deprecated-declarations` flag when compiling source files whose paths 
match the `external/v8` pattern, i.e. V8 source files only.